app/javascript/components/my-toastify.js

```diff
@@ -0,0 +1,27 @@
+import React, {useEffect} from 'react'
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+const MyComponent = ({greeting}) => {
+
+  useEffect(() => {
+    toast(greeting, {
+      position: "top-center",
+      type: "alert",
+      autoClose: 3000,
+      hideProgressBar: false,
+      closeOnClick: true,
+      pauseOnHover: true,
+      draggable: true,
+      progress: undefined,
+      });
```

Comment:
The elements of this JSON can be called from an external file, to avoid overloading this file. What do you think about it?